src/poll.rs

```diff
@@ -58,25 +58,29 @@ pub fn new_registration(poll: &Poll, token: Token, ready: Ready, opt: PollOpt)
 // to verify that a Poll object isn't already registered to a different socket,
 // they use skinny::selector_id(poll).
 //
-// The transmute is obviously dangerous, but the alternative is to build up
-// a lot of machinery that already exists inside Binding and Overlapped (esp.
-// Binding).
+// The cast is obviously dangerous, but the alternative is to build up a lot of
+// machinery that already exists inside Binding and Overlapped (esp. Binding).
 pub mod skinny {
-    use std::mem;
     use std::sync::{atomic::AtomicUsize, Arc, Condvar, Mutex};
     use mio;
 
+    pub fn reinterpret_cast<T, U>(obj: &T) -> &U {
```

Comment:
Doesn't need to be `pub`

Reply:
Good catch. I added that because `mod tests` was originally at file scope, now it's not. Fixing...